requirements.txt

```diff
@@ -3,4 +3,5 @@ Django==4.1.10
 djangorestframework==3.13.1
 pytz==2022.1
 sqlparse==0.4.4
-# mysqlclient==1.4.3
\ No newline at end of file
+mysql-connector-python==8.2.0
+ 
```

Comment:
![image](https://github.com/user-attachments/assets/25d1ceb0-20db-4900-90ef-d7669f28f49a)
Here too, remove that space